.github/RELEASE_NOTES.template.md

```diff
@@ -0,0 +1,13 @@
+# Frequenz Migrogrid API Release Notes
+
+## Summary
+
+<!-- Here goes a general summary of what this release is about -->
+
+## Upgrading
+
+<!-- Here goes notes on how to upgrade from previous versions, including if there are any depractions and what they should be replaced with --> 
+
+## New Features
+
+<!-- Here goes a the main new features and examples or instructions on how to use them -->
```

Comment:
Is it possible to add a section for bug fixes here?

Reply:
I thought these might be details for the extended changelog, but we can add it and if there is nothing that needs special mention, we can remove the section before releasing :+1:

Reply:
The main point here is not just duplicating the whole list of PRs in this section, we should only mention stuff that need some special explanation.

Reply:
Added